README.md

```diff
@@ -29,7 +29,7 @@ ember addon my-addon -b @ember/octane-addon-blueprint
 Included:
 
  - Module Unification
- - Sparkles Components (eventually Glimmer, but built in)
+ - Glimmer Components
  - ember-decorators (eventually will be built in)
```

Comment:
Ember decorators is also no longer a part of the blueprint of could you switch that to 'Native Decorators
And then add 'Tracked Properties'

Thanks!